README.md

```diff
@@ -8,7 +8,7 @@
 
 # Cookie AutoDelete
 
-Control your cookies! This extension is inspired by [Self-Destructing Cookies](https://addons.mozilla.org/en-US/firefox/addon/self-destructing-cookies/). When a tab closes, any cookies not being used are automatically deleted. Prevent tracking by other cookies and add only the ones you trust. Easily import and export your cookie whitelist.
+Control your cookies! This extension is inspired by [Self-Destructing Cookies](https://addons.mozilla.org/firefox/addon/self-destructing-cookies/). When a tab closes, any cookies not being used are automatically deleted. Prevent tracking by other cookies and add only the ones you trust. Easily import and export your cookie whitelist.
```

Comment:
Should probably remove the link entirely at this point since that extension is no longer visible on the site at all.

Reply:
> Should probably remove the link entirely at this point since that extension is no longer visible on the site at all.

Yes, can remove link or use Wayback Machine.  Wayback is still offline though. :(